Lib/test/test_email/test__header_value_parser.py

```diff
@@ -1214,14 +1214,14 @@ def test_get_local_part_valid_and_invalid_qp_in_atom_list(self):
             '@example.com')
         self.assertEqual(local_part.local_part, r'\example\\ example')
 
-    def test_get_local_part_unicode_defect(self):
+    def test_get_local_part_unicode(self):
```

Comment:
This is not, IMO, the best solution.  But fixing it *right* is non-trivial.  The issue here is that this *IS* a defect if policy.utf8=False, and that should be maintained.  The problem is that currently the header value parser does not have access to the policy  Which was a design mistake.

The *reason* you want to maintain the old behavior (aside from backward compatibility, which definitely counts) is that the parser is designed to be used in multiple scenarios, and one of those scenarios might be a place were we want to be emulating a non-SMTPUTF8 parser and looking at where a parsed message does not pass muster.

Now, that said, I'm not seeing any good way to fix this *right*.  I think the best compromise might be to leave this defect alone, but to change the Address object so that it specifically ignores this defect (does not raise an error in that case).  The smtp library should be raising an error if an attempt is made to actually send using a utf8 address to a server that does not support SMTPUTF8.  With this compromise backward compatibility will be maintained, while still allowing utf8 addresses to be specified using Address (which should only be used by library consumers, not the library itself).  I believe I am correct that such addresses will not otherwise be rejected by the email library.

This leaves a dangling issue that if we parse a message using utf8=True we technically should not get a defect, but unless and until we teach the header value parser about the policy we can't really fix that.